crates/gosub_pipeline/viewpage.sh

```
@@ -0,0 +1,6 @@
+#/usr/bin/bash
```

Shellcheck (0.11.0)

[error] Use #!, not just #, for the shebang.

(SC1113)